### ISSUE TITLE
Accessibility: No option to skip repeated content

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#442]
**Description**
All pages on a site will have repeated content. This typically includes logos, navigation, and page titles. Accessibility requires that there is a quick way to skip this repeated content so that users can get to the main content of a page. This is especially important for keyboard navigation and screen readers.

This "skip to content" link should be the first element when a user tabs onto a page. This link should be hidden unless focused on and then will typically show on the top left of a page above other content. WCAG has a [page explaining procedure](https://www.w3.org/TR/WCAG20-TECHS/G1.html) for these types of links.

Example
This is an example that is in use on LibGuides for this type of link.

`<nav aria-label="Skip to main content"><a id="s-lg-public-skiplink" class="alert-info" href="#s-lib-public-main">Skip to main content</a></nav>` - This code is placed at the top of the guide template body.

This styling is added to the link for its default view:
```padding: 10px;
position: absolute;
top: calc(-100%);
left: 0;
background-color: #fff;
border-right: 1px solid #fff;
border-bottom: 1px solid #fff;
border-bottom-right-radius: 8px;
transition: top 1s ease-out,background 1s linear;
z-index: 100;
```

When focused, the "skip to content" link has these styles:
```position: absolute;
left: 0;
top: 0;
transition: top .1s ease-in,background .5s linear;
```

**Solution**
Add a link that will direct users to the div with `role="main"` as the first focusable item on all pages.

**Comment by mhk33** [comment]
There appears to be a skip link in dev/3.0 upgrade.
